.github/workflows/test-sim.yml

```diff
@@ -52,9 +52,15 @@ jobs:
       #   run: yarn test:sim:multiThread
       #   working-directory: packages/beacon-node
       #   env: {RUN_ONLY_SIM_TEST: altair-epoch2}
-      - name: Upload debug log test files
+      - name: Upload debug log test files for "packages/beacon-node"
```

Comment:
Tests must be de-duplicated. This PR should replace the existing sim tests and cover all their cases + augment them by adding assertions.

Reply:
As discussed earlier we will add assertions in multiple PRs. Once we reach to a satisfactory level, we can remove existing sim tests in the beacon-node.